.config/nextest.toml

```diff
@@ -1,5 +1,5 @@
 [profile.default]
-slow-timeout = { period = "60s", terminate-after = 2, grace-period = "0s" }
```

Comment:
FWIW this is not seconds, but rather a multiple of the `period` (which is 60 seconds currently.)

It is unfortunate that we have tests that run for more than two minutes straight, but provided that we don't want to fix the test itself, this looks like a correct way to go about fixing this (especially since we give much more time for the tests in CI...)